nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/TestNuxeoDriveManager.java

```diff
@@ -124,6 +131,13 @@
 
     protected DocumentModel folder_2_1;
 
+    public static class CustomLogFilter implements LogCaptureFeature.Filter {
+        @Override
+        public boolean accept(LogEvent event) {
+            return Level.ERROR.equals(event.getLevel());
+        }
+    }
+
```

Comment:
You can probably use `@LogCaptureFeature.FilterOn(logLevel = "ERROR")` on the test method.